codec-redis/src/main/java/io/netty/handler/codec/redis/BulkStringRedisMessage.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2016 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License, version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at:
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.netty.handler.codec.redis;
+
+import io.netty.buffer.ByteBuf;
+
+/**
+ * Bulk Strings of <a href="http://redis.io/topics/protocol">RESP</a>
+ */
+public interface BulkStringRedisMessage extends RedisMessage {
```

Comment:
@normanmaurer , I and @mp911de didn't add `ReferencedCounted` to it because "null bulk string" and "empty bulk string" don't need to check referenced count. I think we can use `Unpooled.EMPTY_BUFFER` for "empty bulk string" or "null bulk string" if we can distinguish "empty" and "null".